webapp/src/components/modals/channel_settings/select_channel_subscription.tsx

```diff
@@ -54,6 +57,28 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
             );
         }
 
+        let subName = '';
+        if (subscriptionToDelete && subscriptionToDelete.name) {
+            subName = ` "${subscriptionToDelete.name}"`;
+        }
```

Comment:
Same `confirmMessage` suggestion here

Reply:
Not sure I understand what your suggestion is..

Reply:
Hmm I think I forgot to submit the other comment. Basically this:
```
let confirmDeleteMessage = 'Delete Subscription?';
if (subscriptionToDelete && subscriptionToDelete.name) {
    confirmDeleteMessage = `Delete Subscription "${subscriptionToDelete.name}"?`;
}
```

Also in `edit_channel_settings`. I think it's cleaner this way, what do you think?

Reply:
Yep.  I like that suggestion.  Coming out in a few!